src/sass/layouts/_contacts.scss

```diff
@@ -1,6 +1,153 @@
-.our-location{
+.contscts{
+    margin-bottom: 39px;
+    @media screen and(min-width:768px){
+        margin-bottom: 40px;
+    }
+    @media screen and(min-width:1200px){
+        margin-bottom: 120px;
+    }
+}
+.locations{
```

Comment:
Pls add: margin-bottom: 32px; instead of margin-top on buttons (Mobile)

Reply:
updated